src/delagent/ui/admin-upload-delete.php

```diff
@@ -190,6 +193,14 @@ public function Output()
     $text = _("Delete");
     $V.= "<input type='submit' value='$text'>\n";
     $V.= "</form>\n";
+
+    $V.= <<<EOT
```

Comment:
The php statements in heredocs get also executed and replaced. If you want to prevent this use the nowdoc syntax, i.e.
```
    $V.= <<<'EOT'
[...]
```